ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import PropTypes from 'prop-types';
+
+const GraphAlternateView = ({ className, graphData, colNum, title }) => {
+  const columnsOne = [
+    {
+      Header: 'Date',
+      accessor: 'date',
+    },
+    {
+      Header: 'Failure Count per Push',
+      accessor: 'failurePerPush',
+    },
+  ];
+
+  const columnsTwo = [
+    {
+      Header: 'Date',
+      accessor: 'date',
+    },
+    {
+      Header: 'Failure Count',
+      accessor: 'failureCount',
+    },
+    {
+      Header: 'Push Count',
+      accessor: 'pushCount',
+    },
+  ];
+
+  const alternateGraph = {
+    column: [],
+    data: [],
+  };
+
+  graphData.forEach(item => {
+    if (colNum === 1) {
+      alternateGraph.column = columnsOne;
+
+      item.data.forEach(itemOne => {
+        const { date, failurePerPush } = itemOne;
+
+        alternateGraph.data.push({
+          date,
+          failurePerPush,
+        });
+      });
+    } else {
+      alternateGraph.column = columnsTwo;
+
+      item.data.forEach((itemTwo, indexTwo) => {
```

Comment:
I think it's confusing if you name these `itemTwo` and `indexTwo`... just item and index should suffice.